lib/logstorage/rows.go

```diff
@@ -65,6 +65,10 @@ func (f *Field) unmarshal(src []byte) ([]byte, error) {
 type rows struct {
 	fieldsBuf []Field
 
+	// maxUniqueFields is the maximum number of unique fields which may be stored in fieldsBuf.
+	// it is used to perform worst case estimation when merging rows.
+	maxUniqueFields int
```

Comment:
maxUniqueFields looks like a immutable constant for given rows. But in fact, it's a current counter of uniqueField. Should it be renamed?

Reply:
Renamed to make it more clear